src/components/Spinner/index.jsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import './styles.scss';
+
+const Spinner = ({ isAbsolute, size }) => (
+  <div className={`spinner ${isAbsolute ? 'large' : size}`} />
+);
+
```

Comment:
Looks great but can you add PropType validation for this component

Reply:
Thanks @valmayaki for the feedback. let me fix it.